blocks/components/editable/index.js

```diff
@@ -84,3 +84,7 @@ export default class Editable extends wp.element.Component {
 		);
 	}
 }
+
+Editable.defaultProps = {
+	nodeName: 'div'
```

Comment:
There's no `nodeName` prop for `Editable` and an effective default is already assigned in the `render`. We can remove this set of lines.